templates/pages/page.html.twig

```diff
@@ -430,37 +430,33 @@
+                                                            </p>
+                                                        </div>
+                                                    </div>
                                                 <div class="govuk-grid-column-one-half">
                                                     <h2 class="visuallyhidden">You might also be interested in:</h2>
-                                                    <div class="apollo-call-to-action apollo-call-to-action--aggregation">
-                                                        <h3 class="govuk-heading-m">Aggregation</h3>
-                                                        <p>Bulk buying, also known as aggregation,
-                                                            offers highly competitive prices and a
-                                                            simple tender process.</p>
-                                                        <p><a href="/help-and-support/aggregation/">Learn
-                                                                more <span class="visuallyhidden">about aggregation</span></a>
-                                                        </p>
-                                                    </div>
-                                                </div>
-                                            <div class="govuk-grid-column-one-half">
-                                                <h2 class="visuallyhidden">You might also be interested in:</h2>
-                                                    <div class="apollo-call-to-action apollo-call-to-action--help">
-                                                        <h3 class="govuk-heading-m">Can't find what
-                                                            you're looking for?</h3>
-                                                        <p>If none of your requirements can be met by a
-                                                            CCS catalogue or agreement, you can run an
-                                                            EU tender.</p>
-                                                        <p><a href="/help-and-support/eu-tender/">Learn
-                                                                more <span class="visuallyhidden">EU tenders</span></a>
-                                                        </p>
+                                                        <div class="apollo-call-to-action apollo-call-to-action--help">
+                                                            <h3 class="govuk-heading-m">{{ option_cards.second_option_card.title }}</h3>
+                                                            <p>{{ option_cards.second_option_card.description }}</p>
+                                                            <p><a href="{{option_cards.second_option_card.link_url}}">{{ option_cards.second_option_card.link }}<span class="visuallyhidden">EU tenders</span></a>
```

Comment:
Should this still say EU tenders?

Reply:
No, this will be the same as link text so I will change this.